DataFormats/Phase2TrackerDigi/interface/Hit.h

```diff
@@ -0,0 +1,21 @@
+#ifndef HIT_H
+#define HIT_H
+
+class Hit {
```

Comment:
`Hit` looks too generic for such a specific class. Please consider renaming and or using a namespace.

Reply:
Renamed to DigiHitRecord